tracee-ebpf/Makefile

```diff
@@ -127,6 +127,8 @@ endif
 # record built image id to prevent unnecessary building and for cleanup
 docker_builder_file := $(OUT_DIR)/$(DOCKER_BUILDER)
 .PHONY: $(DOCKER_BUILDER)
+# Use Gnu Make 4.3 "grouped targets" to make both $(DOCKER_BUILDER)
+# and $(docker_builder_file) with one command.
```

Comment:
I think it's sufficient to mention this in the readme prereqs as we wouldn't want to document every time we use this feature of other "advanced" features. perhaps mention as an FAQ in github discussions. Do you think that would suffice?
(applies to other occurrence as well)

Reply:
I am happy to remove this comment, or even to rewrite the Makefile to not require the feature. I haven't even decided in my own mind if it makes sense to have a .PHONY and a real target as "grouped targets".

Reply:
maybe just put a comment at the top of the file that this makefile required GNU Make 4.3, I think that's will help.

> rewrite the Makefile to not require the feature

happy to hear about ideas for improvements! If you have a concrete idea, please open an issue do discuss it